utils/generate-docs/Schema/SchemaReader.ts

```diff
@@ -23,9 +23,9 @@ export default class SchemaReader {
   protected loadJsonFromEntryInfo = (
     entryInfo: EntryInfo
   ): Promise<SchemaNodeJson> =>
-    import(pathToFileURL(entryInfo.fullPath).href).then(
-      (json: { default: SchemaNodeJson }) => json["default"]
-    );
+    import(pathToFileURL(entryInfo.fullPath).href, {
```

Comment:
This change is throwing a rather cryptic syntax error for me (and is breaking the test too):

```console
jman@jman-Blade-15-Advanced-Model-Mid-2021-RZ09-0409:~/source/Open-Cap-Format-OCF$ npm run generate-docs

> generate-docs
> node --loader ts-node/esm --no-warnings --experimental-json-modules ./utils/generate-docs/index.ts

SyntaxError: Unexpected token ','
    at ESMLoader.moduleStrategy (node:internal/modules/esm/translators:139:18)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:236:14)
```

Based on what you added, my guess is one of our jsons has a trailing comma or something that one validator likes and another doesn't or something like that? Your change to `PathTools.ts` works. I'm inclined to drop the assertions and move resolving whatever is causing this error to a separate, low priority task.

Reply:
Just saw #386 . Fix here is upgrading to node v16.16.0

Reply:
Node 16 is almost end of life. Security support ends September of this year. We should upgrade to 18 LTS.